src/Application.vala

```diff
@@ -190,6 +186,31 @@ public class Mail.Application : Gtk.Application {
         main_window.present ();
     }
 
+    public async void request_background () {
+        var portal = new Xdp.Portal ();
+
+        Xdp.Parent? parent = active_window != null ? Xdp.parent_new_gtk (active_window) : null;
+
+        var command = new GenericArray<weak string> ();
+        command.add ("io.elementary.mail");
+        command.add ("--background");
+
+        try {
+            if (!yield portal.request_background (
+                parent,
+                _("Mail needs to autostart and run in background in order to send notifications even while closed."),
```

Comment:
I think we shouldn't be relying on the app to name itself or the permissions its requesting. That should be the responsibility of the portal. So we should only have here the reason like:
```suggestion
                _("Send notifications when new mail arrives."),
```

Reply:
It seems like we aren't in charge of that yet, so editing my suggestion